tests/flow/annot/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -163,38 +131,6 @@ function foobar(n: ?number): number | null | void {
 function barfoo(n: number | null | void): ?number {
   return n;
 }
-
-type Banana = {
```

Comment:
Those are tests that we manually added in the wrong place. Would be nice to restore them back outside of the flow folder.